Makefile

```diff
@@ -1,3 +1,7 @@
+REGISTRY         = docker.io/ansibleplaybookbundle
+TAG              = latest
```

Comment:
Instead of setting these values, let's set them as defaults and let the user override these vars by doing: ```make apb-images TAG=old```. This would look like: ```TAG ?= latest```.